cavern/src/intTest/java/org/opencadc/cavern/TransferRunnerTest.java

```diff
@@ -254,6 +300,14 @@ public void testPullFromVOSpace() {
                 }
                 Assert.assertTrue("one or more endpoints for " + reqP.getUri(), (num > 0));
             }
+            
+            // check the file content
+            trans.setInputStreamWrapper(new InputStreamWrapper() {
+                public void read(InputStream inputStream) throws IOException {
+                    byte[] buf = new byte[100];
+                    Assert.assertEquals(3, inputStream.read(buf, 0, 100));
+                }});
+            trans.runTransfer();
```

Comment:
No actual assert that the bytes received are correct, contrary to the comment. To pass it has to run without failing so that is most of the test. I would declare buf as final outside the anon class and add an assert after the run